### PR TITLE
Documentation/v2: 'etcd v2' to the title

### PR DESCRIPTION
**Pull request by gyuho**
Fix https://github.com/coreos/etcd/issues/8152.

### CODE REVIEW

File: Documentation/v2/04_to_2_snapshot_migration.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Snapshot Migration
```
Comment (xiang90) [inline Documentation/v2/04_to_2_snapshot_migration.md]: probably add a header is good enough?

something like **This is the documentation for etcd2 releases. Read [etcd3 doc][v3] for etcd3 releases**

**Comment by gyuho** [comment]
@xiang90 All fixed. PTAL.

**Comment by xiang90** [comment]
lgtm if ci passes.

**Comment by gyuho** [comment]
CI failure not related. marker, Document style check all passed.

**Pull request merged**